squire_web/src/tournament/players/mod.rs

```diff
@@ -27,6 +29,7 @@ pub struct PlayerFilterProps {
 pub enum PlayerFilterMessage {
```

Comment:
I know that I implemented this, but the `PlayerView` struct used to be called `PlayerFilter`. I renamed it when doing some code clean up, but `PlayerFilterProps` and `PlayerFilterMessage` should probably be renamed to `PlayerViewX`.